CHANGELOG.md

```diff
@@ -61,6 +61,17 @@ Adding a new version? You'll need three changes:
  - [0.0.5](#005)
  - [0.0.4 and prior](#004-and-prior)
 
+## [2.9.0]
+
+> Release date: TBD
+
+### Fixed
+
+- When `CombinedRoutes` is turned on, translator will replace appearance of
+ `*` in host of `Ingress` to `_` in kong route names because `*` is not 
```

Comment:
```suggestion
- When `CombinedRoutes` is turned on, translator will replace each occurrence of
 `*` in `Ingress`'s host to `_` in kong route names because `*` is not 
```